AzureMonitorAgent/agent.py

```diff
@@ -442,6 +442,14 @@ def enable():
     ssl_cert_var_name, ssl_cert_var_value = get_ssl_cert_info('Enable')
     default_configs[ssl_cert_var_name] = ssl_cert_var_value
 
+    _, _, _, az_environment, _ = me_handler.get_imds_values(is_lad=False)
+    if az_environment.lower() == me_handler.ArcACloudName:
+        _, mcs_endpoint = me_handler.get_arca_endpoints_from_himds()
+        default_configs["customRegionalEndpoint"] = mcs_endpoint
+        default_configs["customGlobalEndpoint"] = mcs_endpoint
+        default_configs["customResourceEndpoint"] = "https://monitoring.azs"
```

Comment:
this url seems incomplete

Reply:
Hi @simathih , for ArcA (an edge zone environment), there's only one MCS endpoint, like https://amcs.monitoring.autonomous.cloud.private/ . These url here are correct for ArcA env. And we have tested on ArcA environment, AMA ext could be installed and metrics could be ingested successfully.

![image](https://github.com/Azure/azure-linux-extensions/assets/56957158/3437e612-231a-448e-9663-31a5594ad0e2)